releasenotes/notes/add-primitives-support-31af39549b5e66e3.yaml

```diff
@@ -0,0 +1,24 @@
+---
+prelude: >
+    Qiskit Optimization 0.5 supports the new algorithms introduced in Qiskit Terra 0.22
+    which in turn rely on the `Qiskit Primitives <https://qiskit.org/documentation/apidoc/primitives.html>`_.
+    Qiskit Optimization 0.5 still supports the former algorithms based on ``QuantumInstance``,
+    but they will be deprecated and then removed, along with the support here, in future releases.
+
+features:
+  - |
+    The :class:`~.MinimumEigenOptimizer` class takes the primitives-based algorithms
+    (``qiskit.algorithms.minimum_eigensolvers.SamplingMinimumEigensolver`` and
+    ``qiskit.algorithms.minimum_eigensolvers.NumPyMinimumEigensolver``)
+    as ``min_eigen_solver`` argument.
+    The former algorithm ``qiskit.algorithms.minimum_eigen_solvers.MinimumEigensolver``
+    is pending deprecation and will be deprecated and subsequently removed in future releases.
+    Note that ``qiskit.algorithms.minimum_eigensolvers.SamplingVQE`` supersedes
+    ``qiskit.algorithms.VQE`` for :class:`~.MinimumEigenOptimizer`.
+    ``qiskit.algorithms.minimum_eigensolvers.NumPyMinimumEigensolver`` also supersedes
+    ``qiskit.algorithms.minimum_eigen_solvers.NumPyMinimumEigensolver`.
```

Comment:
```suggestion
    ``qiskit.algorithms.minimum_eigen_solvers.NumPyMinimumEigensolver``.
```
Perhaps the release note is failing due to the mistmatch here - starts with \`\` but ended with a single \`

Reply:
Thanks. You are right.